File: Math/1317. Convert Integer to the Sum of Two No-Zero Integers.cpp
class Solution {
public:
    vector<int> getNoZeroIntegers(int n)
    {
        for(int i = 1; i < n; i++)
        {
            string num1 = to_string(i);
            string num2 = to_string(n - i);
            if(num1.find('0') == -1 and num2.find('0') == -1)
                return {i, n - i};
        }
        return {0};
    }
};

File: 1317. Convert Integer to the Sum of Two No-Zero Integers.py
class Solution:
    def getNoZeroIntegers(self, n: int) -> List[int]:
        loop = int(n / 2) + 2
        for i in range(loop):
            if '0' in str(i) or '0' in str(n - i):
                continue
            return [i, n - i]
        return None